pkg/action/release_testing.go

```diff
@@ -110,13 +111,27 @@ func (r *ReleaseTesting) GetPodLogs(out io.Writer, rel *release.Release) error {
 	for _, h := range rel.Hooks {
 		for _, e := range h.Events {
 			if e == release.HookTest {
-				req := client.CoreV1().Pods(r.Namespace).GetLogs(h.Name, &v1.PodLogOptions{})
+				podLogOptions := &v1.PodLogOptions{}
+
+				pod, err := client.CoreV1().Pods(r.Namespace).Get(context.Background(), h.Name, metav1.GetOptions{})
+				if err != nil {
+					return errors.Wrapf(err, "unable to get pod info for %s", h.Name)
+				}
+
+				if container, ok := pod.Annotations["helm.sh/hook-logs-container"]; ok {
```

Comment:
I suggest using the well known annotation [kubectl.kubernetes.io/default-container](https://kubernetes.io/docs/reference/labels-annotations-taints/#kubectl-kubernetes-io-default-container)